runtime/ops/permissions.rs

```diff
@@ -90,11 +142,33 @@ pub fn op_revoke_permission(
   state: &mut OpState,
   #[serde] args: PermissionArgs,
 ) -> Result<PermissionStatus, AnyError> {
-  let mut permissions = state.borrow_mut::<PermissionsContainer>().0.lock();
+  // todo(dsherret): don't have this function use the properties of
+  // permission container
+  let permissions_container = state.borrow_mut::<PermissionsContainer>();
```

Comment:
What's wrong with that?

Reply:
It's exposing a mutex, it's having code that shouldn't be done/exposed at this level (parsing descriptors), and it's coupling the private api to this part of the code.

Reply:
Open an issue to keep track of it? Or are you planning to address it before landing?

Reply:
https://github.com/denoland/deno/issues/25634